src/dissolve-gui-qml.cpp

```diff
@@ -0,0 +1,101 @@
+// SPDX-License-Identifier: GPL-3.0-or-later
+// Copyright (c) 2024 Team Dissolve and contributors
+
+#include "base/messenger.h"
+#include "base/processPool.h"
+#include "gui/gui.h"
```

Comment:
```suggestion
#include "gui/models/dissolveModel.h"
```

We don't want to include the entire GUI, but only the main model.  I mention this mostly because it enables us to skip linking against some dependencies that we would like to escape.

Reply:
Makes sense!